### ISSUE TITLE
macOS arm machines not supported

### ISSUE THREAD

**Issue by ipinky77** [wilix-team/iohook#378]
## Expected Behavior
arm Mac machines not supported

## Current Behavior

npm ERR! Prebuild for current platform (iohook-v0.9.3-node-v93-darwin-arm64) not found!
npm ERR! Try to build for your platform manually:

## Steps to Reproduce (for bugs)

npm install iohook on Mac arm machine

**Comment by marcelblum** [comment]
It does work if you build it manually. Just did it earlier today :) Clone the full repo, `npm install`, then `node build.js` for your desired target as detailed at https://wilix-team.github.io/iohook/manual-build.html

**Comment by ondoki** [comment]
This does not work like this , since the regular "npm install" triggers the install.js File which tries to download a prebuild and throws an exception. A build works but throws an exception if you try to add it to the project modules. Could you describe all the steps you did specifically. Thank you :-)

**Comment by ondoki** [comment]
Yeah I got it working by cloning the iohoook repo into my project node_modules , then doing a npm install which fails , but afterwords I still can build the project via npm run build. Then I create the builds directory and within a node-v93-darwin-arm64 folder  >  create within a Release Folder  and paste in both the iohook.node and the uihook,dylib. But this feels a little bit hacky and I probably will try to write a napi wrapper myself --> Not sure if this here is still maintained

**Comment by ondoki** [comment]
But thank you

**Comment by marcelblum** [comment]
@ondoki that's one way to do it. The reason `npm install` fails is because you are running an arch/version of Node not yet officially supported by iohook. The install script autodetects your platform and tries to download the nonexistent prebuild which fails. You could avoid the errors by specifying a supported version in your project's package.json as specified at https://wilix-team.github.io/iohook/usage.html, then at least it would find the prebuild and install without throwing an exception. And yes it is hacky but it works fine once built 😃
